framework/python/src/common/session.py

```diff
@@ -38,13 +38,39 @@
 CERTS_PATH = 'local/root_certs'
 CONFIG_FILE_PATH = 'local/system.json'
 SECONDS_IN_YEAR = 31536000
+STATUS_TOPIC = 'status'

```

Comment:
Should it be /events/status?

Reply:
According to notes from Bi-weekly sync, it was proposed to call this topic "status"

Reply:
Correct yes, as this is not necessarily an event, it should just be could /status.